nautobot/dcim/tests/test_api.py

```diff
@@ -2140,9 +2140,10 @@ def test_remove_chassis_from_master_device(self):
         virtual_chassis_1 = response.json()["results"][0]
 
         # Make sure the master is set
-        self.assertNotEqual(virtual_chassis_1["master"], None)
+        self.assertIsNotNone(virtual_chassis_1["master"])
 
-        master_device = Device.objects.get(pk=virtual_chassis_1["master"])
+        # The `master` key will be a URL now, but it contains the PK
+        master_device = Device.objects.get(pk=virtual_chassis_1["master"].split("/")[-2])
```

Comment:
TODO: helper function to extract a pk from an api url or even return the instance of the object?